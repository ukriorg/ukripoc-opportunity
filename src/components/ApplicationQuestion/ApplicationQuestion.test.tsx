import React from "react";
import renderer from "react-test-renderer";
import { ApplicationQuestion } from "./index";

describe("ApplicationQuestion", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<ApplicationQuestion />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});

