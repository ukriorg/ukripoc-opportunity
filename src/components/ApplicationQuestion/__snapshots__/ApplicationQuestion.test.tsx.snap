// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationQuestion renders correctly 1`] = `
<h1
  className="sc-kAzzGY jDbFAD sc-cSHVUG bVSTLB"
  size="XLARGE"
>
  Loading...
</h1>
`;
